test/client/spec/components/victory-candlestick/victory-candlestick.spec.js

```diff
@@ -0,0 +1,149 @@
+      expect(svg.prop("style").height).to.equal("auto");
+    });
+
+    it("renders an svg with the correct viewBox", () => {
+      const wrapper = mount(
+        <VictoryCandlestick data={dataSet}/>
+      );
+      const svg = wrapper.find("svg");
+      const viewBoxValue =
+        `0 0 ${VictoryCandlestick.defaultProps.width} ${VictoryCandlestick.defaultProps.height}`;
+      expect(svg.prop("viewBox")).to.equal(viewBoxValue);
+    });
+
+    it("renders 51 points", () => {
+      const wrapper = shallow(
+        <VictoryCandlestick/>
+      );
+      const points = wrapper.find(Candle);
+      expect(points.length).to.equal(51);
+    });
+  });
+
+  describe("rendering data", () => {
+    it("renders injected points for {x, y} shaped data (default)", () => {
+      const data = range(10).map((i) => ({x: i, y: i}));
+      const wrapper = shallow(
+        <VictoryCandlestick data={data} dataComponent={<MyCandle />} />
+      );
+
+      const points = wrapper.find(MyCandle);
+      expect(points.length).to.equal(10);
+    });
+
+    it("renders points for {x, y} shaped data (default)", () => {
+      const data = range(10).map((i) => ({x: i, y: i}));
+      const wrapper = shallow(
+        <VictoryCandlestick data={data}/>
+      );
+      const points = wrapper.find(Candle);
+      expect(points.length).to.equal(10);
+    });
+
+    it("renders points for array-shaped data", () => {
+      const data = range(20).map((i) => [i, i]);
+      const wrapper = shallow(
+        <VictoryCandlestick data={data} x={0} y={1}/>
+      );
+      const points = wrapper.find(Candle);
+      expect(points.length).to.equal(20);
+    });
+
+    it("renders points for deeply-nested data", () => {
+      const data = range(40).map((i) => ({a: {b: [{x: i, y: i}]}}));
+      const wrapper = shallow(
+        <VictoryCandlestick data={data} x="a.b[0].x" y="a.b[0].y"/>
+      );
+      const points = wrapper.find(Candle);
+      expect(points.length).to.equal(40);
+    });
+
+    it("renders data values with null accessor", () => {
+      const data = range(30);
+      const wrapper = shallow(
+        <VictoryCandlestick data={data} x={null} y={null}/>
+      );
+      const points = wrapper.find(Candle);
+      expect(points.length).to.equal(30);
+    });
+  });
+
+  describe("event handling", () => {
+    it("attaches an event to data", () => {
+      const clickHandler = sinon.spy();
+      const wrapper = mount(
+        <VictoryCandlestick
+          data={dataSet}
+          events={[{
+            target: "data",
+            eventHandlers: {onClick: clickHandler}
+          }]}
+        />
+      );
+      const Data = wrapper.find(Candle);
+      Data.forEach((node, index) => {
+        const initialProps = Data.at(index).props();
+        node.simulate("click");
+        expect(clickHandler.called).to.equal(true);
+        // the first argument is the standard evt object
+        expect(omit(clickHandler.args[index][1], ["events", "key"]))
```

Comment:
Try commenting out the second two expect statements and see if the clickHandler is even called. After that I would try `console.log`ing the clickHandler.args[index]. deep `eql` is pretty unforgiving.